cloud-run/fhir-converter/main.py

```diff
@@ -1,16 +1,37 @@
 from pathlib import Path
-from typing import Union, Optional
 import subprocess
 import json
 
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+api = FastAPI()
+
+
+class FhirConverterInput(BaseModel):
+    """
+    Input parameters for the FHIR Converter.
+    """
+
+    input_data: str
+    input_type: str
+    root_template: str
```

Comment:
I really like this use of Pydantic, it actually inspired me to get something similar going for the `upload_fhir_bundle` function. Could we add some additional specificity/validation here? For example, while 'input_data' will be a string it should also be valid JSON. Also, can probably treat `input_type` as an enum. We might be able to do the same for `root_template` but we will need to think about templates being added and removed in the future.

Reply:
I like the idea of using enums, I'll add that in! But is it true that `input_data` should be valid JSON? The [sample input HL7 data I used](https://github.com/microsoft/FHIR-Converter/blob/main/data/SampleData/Hl7v2/ADT-A01-01.hl7) looks like pipe delimited data?

Reply:
Yes, you are totally right! I was still lost thinking in terms of the upload function which takes FHIR as input. Our input data here with either being pipe delimitated Hl7 or XML (CCDA for the ECRs).